### PR TITLE
feat: typography and colors for /protocol

### PR DESCRIPTION
**Pull request by ryan-ludwig**
https://test.kiva.org/protocol
https://www.kiva.org/protocol

### CODE REVIEW

File: src/pages/Protocol.vue [hunk 1]
```diff
@@ -1,115 +1,32 @@
...
 					{{ introTitle }}
-				</div>
+				</p>
 				<p v-for="(paragraph, index) in introText" :key="index">
 					{{ paragraph }}
 				</p>
 				<p><a href="https://pages.kiva.org/kiva-protocol-faq">{{ introLinkTitle }}</a></p>
 			</div>
 		</div>
-		<div v-if="showPartners" class="row" id="partners">
```
Comment (ryan-ludwig) [inline src/pages/Protocol.vue:24]: This block has been turned off for 3 years

File: src/pages/Protocol.vue [hunk 2]
```diff
@@ -1,115 +1,32 @@
...
-						</div>
-					</div>
-				</div>
-			</div>
-			<div id="partner-become" class="small-12 columns">
-				<!-- eslint-disable-next-line max-len -->
-				Interested in becoming a partner? Email us at <a href="mailto:contactus@kiva.org?subject=Kiva Protocol partner">contactus@kiva.org</a>
-			</div>
-		</div>
-		<div v-if="showPress" id="press-container">
```
Comment (ryan-ludwig) [inline src/pages/Protocol.vue:73]: This block has been turned off for 3 years

File: src/pages/Protocol.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/Protocol.vue:24]: Yea pretty sure we turned it off the first week lolz...

**Review by mcstover — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
👍   great!

**Pull request merged**